src/components/recordset/recordset-table.tsx

```diff
@@ -472,10 +507,10 @@ const RecordsetTable = ({
     const tableSchemaNames = `s_${makeSafeIdAttr(reference.table.schema.name)} t_${makeSafeIdAttr(reference.table.name)}`;
     return classNameString + ' ' + tableSchemaNames;
   }
-
   return (
     <div className='recordset-table-container' ref={tableContainer}>
-      <div className='chaise-table-top-scroll-wrapper'>
+      <div ref={stickyScrollbarRef}
+        className={`chaise-table-top-scroll-wrapper ${isBottomVisible ? 'no-scroll-bar' : ''}`}>
```

Comment:
Better to write this like:
```
`chaise-table-top-scroll-wrapper${isBottomVisible ? ' no-scroll-bar' : ''}`
```
so that the _whitespace_ is only added if the extra class is also added.